docs/api/events.rst

```diff
@@ -865,6 +865,18 @@ Members
     :param after: The updated member's updated info.
     :type after: :class:`Member`
 
+.. function:: on_raw_presence_update(payload)
+
+    Called when a member updates their presence.
+    Unlike :func:`on_presence_update`, this is called regardless of the member cache.
```

Comment:
```suggestion
    Unlike :func:`on_presence_update`, this is called regardless of the member cache.
    
    Since the data payload can be partial and the Discord API does not validate the types of the fields,
    care must be taken when accessing stuff in the dictionary.
```
additional disclaimer to be sure; phrasing mostly copied from `on_raw_message_edit`